app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrations.jsx

```diff
@@ -79,7 +79,7 @@ export const Integrations = () => {
   };
 
   return (
-    <>
+    <div style={{ maxWidth: '1200px', margin: '0 auto' }}>
```

Comment:
The same as in https://github.com/reportportal/service-ui/pull/3361.
Follow the recommendations that @AmsterGet provided there.